Logstash/logstash-input-azurewadtable/lib/logstash/inputs/azurewadtable.rb

```diff
@@ -52,14 +53,34 @@ def run(output_queue)
   def teardown
   end  
 
-  def process(output_queue)
-    @logger.debug(@last_timestamp)
+  def build_latent_query
```

Comment:
partitionkey_from_datetime omits seconds from time. So, if the difference between until_timestamp and last_timestamp is less than 1 minute, build_latent_query behaves like build_zero_latency_query.

As a result, there could be the case when data from other VMs will be lost:

1. last_timestamp = 06:59, data_latency_seconds = 1
2. VM-A wrote data at 07:00:00.000Z
3. logstash queried data at 07:00:14.000Z. until_timestamp is 07:00:13.000Z. The query is: 06:59 < PartitionKey < 07:00. 
Got data from VM-A, moved last_timestamp to 07:00
4. VM-B wrote data at 07:00:15.000Z
5. logstash queried data at 07:00:29.000Z. until_timestamp is 07:00:28.000Z. The query is:  07:00 < PartitionKey < 07:00.

Data from VM-B is lost.

1. data_latency_seconds fraction should be minutes.
2. Is there a scenario when someone need more than minute delay? Why would someone need to configure data_latency_seconds at all?